read-the-docs/source/JIRA.rst

```diff
@@ -25,6 +26,20 @@ Methods
 -  `add_watcher() <#add_watcher>`__
 -  `add_attachment() <#add_attachment>`__
 
+get_ticket_content()
+--------------------
+
+``get_ticket_content(self, ticket_id=None)``
+
+Queries the JIRA API to get ticket_content using ticket_id. The ticket_content
+is expressed in a form of dictionary as a result of JIRA API get product:
```

Comment:
I think you mean:
"as a result of JIRA API's get issue"

Reply:
Changed.